chatkit-core/src/main/kotlin/com/pusher/chatkit/users/UserSubscriptionEvent.kt

```diff
@@ -9,7 +9,7 @@ internal typealias UserSubscriptionConsumer = (UserSubscriptionEvent) -> Unit
 internal sealed class UserSubscriptionEvent {
 
     internal data class InitialState(
-        val currentUser: User,
+//        val currentUser: User,
```

Comment:
we should have UserApiType for that

Reply:
good spot - i don't think I had a UserApiType at that point - have refactored this part to use the UserApiType.